Dockerfile.ubuntu20

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+
+# Build ubuntu20-based container images for use when building EDK2:
+# - build.  This image has the basic set of tools required to build EDK2.  It's
+# appropriate for use in CI pipelines and other automation.
+# - dev.  This image is the build image, plus a few developer-friendly
+# additions.  It adds more packages and sets an entrypoint to run as the
+# development user.
+
+
+#####################################################################
+# Build Image
+#
+FROM ubuntu:20.04 AS build
+
+# Set the EDKREPO URL (and version)
+ENV EDKREPO_URL=https://github.com/tianocore/edk2-edkrepo/releases/download/edkrepo-v2.1.2/edkrepo-2.1.2.tar.gz
+
+# Suppresses a debconf error during apt-get install.
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Set timezone.
+ENV TZ=America/Denver
```

Comment:
To keep this universal, maybe use UTC here?

Reply:
Sounds good to me.